### ISSUE TITLE
Ensnare problem incl. vid (BB #1101)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#1101]
This issue was migrated from bitbucket.
**Original Reporter:** Dikkedeur
**Original Date:** 15.08.2015 06:25:10 GMT+0000
**Original Priority:** major
**Original Type:** bug
**Original State:** resolved
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1101

<hr>

problem: If the mob targets another player and gets ensnared it teleports.

proof: https://vid.me/dGf9

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** HEGladios
**Original Date:** 15.08.2015 17:06:23 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1101/#comment-20940686

<hr>

The movement generators still have some old bugs looks like.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 15.08.2015 17:08:36 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1101/#comment-20940694

<hr>

Indeed. Do you guys have a solution for this?

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** HEGladios
**Original Date:** 15.08.2015 17:13:59 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1101/#comment-20940774

<hr>

I think TC has very similar coding.
Although maybe Mathman has worked on this? Since he added the spline system a while back.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 26.08.2015 12:56:54 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1101/#comment-21256956

<hr>

This is definitely related to our new movement system. I've got a solution to this one that seems to be working and will push it in the following days.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** HEGladios
**Original Date:** 26.08.2015 13:27:48 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1101/#comment-21257873

<hr>

Awesome.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 26.08.2015 17:29:00 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1101/#comment-21264005

<hr>

[Core/Movement] Prevent removing IGNORE_PATHFINDING unit state on spawn.
                Correctly call StopMoving on root.
                Add some safety checks to confirm paths in Fleeing and Targeted movegens.
                Adjust random movement behaviour. Things will look more smoothly now.
                Add possibility to have formations with random movements.
                Check LoS for targeted movegen.

```
            General cleanups in some movement related code.

            Fixes #1101
```

→ <<cset ecd9aac7dd03>>
